Surface_mesh_parameterization/examples/Surface_mesh_parameterization/CMakeLists.txt

```diff
@@ -38,7 +36,7 @@ if(TARGET CGAL::Eigen3_support)
       message(STATUS "NOTICE: The example `orbifold.cpp` will be compiled without the Sparsesuite library and UmfPack. Try setting SuiteSparse_UMF_INCLUDE_DIR and at least one of SuiteSparse_UMFPACK_LIBRARY_RELEASE and SuiteSparse_UMFPACK_LIBRARY_DEBUG to you UMFPACK installation.")
     endif()
   else(SuiteSparse_FOUND)
-    message(STATUS "NOTICE: The example `orbifold.cpp` will be compiled without the Sparsesuite library.")
+    message(STATUS "NOTICE: The example `orbifold.cpp` will not be compiled without the Sparsesuite library.")
```

Comment:
Sparsesuite -> SuiteSparse ?

Reply:
I fixed a few typos inb cf31d6ed65caa70038a8655864c9a20806f0dd6. There is still one typo in the name of the macro `CGAL_SMP_USE_SPARSESUITE_SOLVERS` but I do not know what I may break if I rename it. It seems an internal detail.

Reply:
If it is not documented I would just change it.